perf-measures/type-check/.octocov.perf-measures.yml

```diff
@@ -0,0 +1,15 @@
+locale: "en"
+repository: ${GITHUB_REPOSITORY}/perf-measures
+coverage:
+  if: false
+codeToTestRatio:
+  if: false
+testExecutionTime:
+  if: false
+diff:
+  datastores:
+    - artifact://${GITHUB_REPOSITORY}
+comment:
+  if: is_pull_request
```

Comment:
Delete this line if you do not need it to appear as a comment in every Pull Request.
You can also check the summary only.

Reply:
How do you think we should enable or disable it? It's cool that the performance measuring result is posted to PR as a comment, but currently, it is only a TypeScript type performance. Many PRs are not related to the type definition, so it may not be important for them.

CC: @m-shaka

Reply:
People often ignore such an automated comment, so it may not be a problem

Reply:
We believe that the comment display itself should always be displayed when measuring performance. Displaying comments makes people aware of performance.

> Many PRs are not related to the type definition, so it may not be important for them.

I thought that the essential answer to this question is that we should consider not nudging the `perf-measures-type-check-on-pr` Job for PRs that do not affect TypeScript type performance.

It is possible to specify a path base for the execution condition of a GHA job, but is the following the only files that affect TypeScript type performance?

* src/client/*.ts
* perf-measures/**/*
If the source itself that measures performance is changed, it is also necessary to run the job

Reply:
@m-shaka @k2tzumi Thank you for the comment!

> We believe that the comment display itself should always be displayed when measuring performance. Displaying comments makes people aware of performance.

I see!

> It is possible to specify a path base for the execution condition of a GHA job, but is the following the only files that affect TypeScript type performance?

We have to add the following files that affect application type definitions:

* `src/types.ts`
* `src/hono-base.ts`
* `src/request.ts`
* `src/context.ts`
* `src/validator/validator.ts` - it can be used in the RPC-mode.

@m-shaka 

What do you think of this?

Reply:
Is it possible that the list of files affecting the application type definition changes frequently? Also, is it difficult to identify these files?

Currently, the performance measurement task takes a sufficiently short time to complete. Therefore, it may be beneficial to run it every time to facilitate maintenance.

Reply:
If the bundle size of https://github.com/honojs/hono/pull/3661 is also subject to comparison output, it might be simpler to run the job each time, but what do you think?

Reply:
> If the bundle size of https://github.com/honojs/hono/pull/3661 is also subject to comparison output, it might be simpler to run the job each time, but what do you think?

Agree! Let's make it simple by running the job each time.